### PR TITLE
Add simple continuous batching benchmark

### PR DESCRIPTION
**Pull request by kdamaszk**
# What does this PR do?

Adds a simple benchmark, which allows to send multiple requests into TGI simultaneously and present basic performance data with continuous batching.

### CODE REVIEW

**Comment by kdamaszk** [comment]
@libinta @regisss please take a look on this. Any feedback is very important

File: examples/README.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
+# TGI-Gaudi example
+
+This example provide a simple way of usage a `tgi-gaudi` with continuous batching. It uses a small dataset [DIBT/10k_prompts_ranked](https://huggingface.co/datasets/DIBT/10k_prompts_ranked) and presetn basic performance numbers.
```
Comment (regisss) [inline examples/README.md]:
```suggestion
This example provide a simple way of usage of `tgi-gaudi` with continuous batching. It uses a small dataset [DIBT/10k_prompts_ranked](https://huggingface.co/datasets/DIBT/10k_prompts_ranked) and present basic performance numbers.
```

File: examples/README.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+### Install
+
+```
+pip install -r requirements
+```
+
+### Setup TGI server
+
+More details on runing the TGI server available [here](https://github.com/kdamaszk/tgi-gaudi/blob/habana-main/README.md#running-tgi-on-gaudi).
```
Comment (regisss) [inline examples/README.md]:
```suggestion
More details on runing the TGI server available [here](https://github.com/huggingface/tgi-gaudi/blob/habana-main/README.md#running-tgi-on-gaudi).
```

**Review by regisss — APPROVED** [review]
LGTM!
@kdamaszk Should I merge it now?

**Comment by kdamaszk** [comment]
Let's wait for @libinta review

**Review by libinta — APPROVED** [review]

**Pull request merged**